### PR TITLE
Web user lookup tweaks

### PR DESCRIPTION
**Pull request by czue**
@snopoke / anyone

(related to, but doesn't fix http://manage.dimagi.com/default.asp?221246)

### CODE REVIEW

File: submodules/couchlog-src [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 4d53524b0e8312b7df685d66b41f4f195e8bc6a9
```
Comment (snopoke) [inline submodules/couchlog-src]: Is this intentional? Seems unrelated to the changes

Comment (czue) [inline submodules/couchlog-src]: nope, accident. https://github.com/dimagi/commcare-hq/commit/081d192b6751fdff08471fc60efd3bd9b3dc854e
